deploy/buildtools/buildconfig.py

```diff
@@ -59,6 +61,8 @@ def __init__(self,
         self.DESIGN = recipe_config_dict['DESIGN']
         self.TARGET_CONFIG = recipe_config_dict['TARGET_CONFIG']
         self.deploytriplet = recipe_config_dict['deploy_triplet']
+        self.frequency = recipe_config_dict['frequency']
```

Comment:
This is more of an @abejgonzalez question since it overlaps with the schema stuff, but you should probably check the string is formatted such that the aws script will accept it, and reject malformed strings immediately.

Reply:
So the getters would return pre-validated values.

Reply:
For now since the schema stuff isn't merged in lets validate the strings here.